ra/ra.go

```diff
@@ -778,24 +778,29 @@ func (ra *RegistrationAuthorityImpl) checkLimits(ctx context.Context, names []st
 }
 
 // UpdateRegistration updates an existing Registration with new values.
-func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (reg core.Registration, err error) {
-	base.MergeUpdate(update)
+func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (core.Registration, error) {
+	if changed := base.MergeUpdate(update); !changed {
```

Comment:
I kind of like having the explicit `changed` variable. I think it makes it easier to read without needing to know the internals of `MergeUpdate`. That said, I'm certainly open to dropping it and using `!base.MergeUpdate(update)` if that's the majority preference.